README.md

```diff
@@ -1,124 +1,89 @@
 # dojot History service
 
 This service handles all operations related to persisting and retrieving historical device
-data.
+data. For more information about the usage, check the [API documentation](https://dojot.github.io/history/apiary_latest.html).
+
+## Dependencies
+
+- falcon
+- gunicorn
+- gevent
+- python-dateutil
+- pymongo
+- requests
+
+The setup of these dependencies are described in the following section.
+
+## Configuration
+
+In order to properly start History, the following variables must be set. All
+values are the default ones, and they need to be modified to the services' address.
+
+Key                         | Purpose                                                      | Default Value
+----------------------------|--------------------------------------------------------------|----------------
+HISTORY_DB_ADDRESS          |History database's address                                    |"mongodb"
+HISTORY_DB_PORT             |History database's port                                       |27017
+HISTORY_DB_REPLICA_SET      |History database's replica set address                        |None
+HISTORY_DB_DATA_EXPIRATION  |Seconds before removing an entry in MongoDB                   |604800
+MONGO_SHARD                 |                                                              |False
+AUTH_URL                    |Auth url address                                              |"http://auth:5000"
+KAFKA_ADDRESS               |Kafta address                                                 |"kafka"
+KAFKA_PORT                  |Kafka port                                                    |9092
+KAFKA_HOSTS                 |KAFKA_ADDRESS concatenated with KAFKA_PORT                    |"kafka:9092"
+KAFKA_GROUP_ID              |Group ID used when creating consumers                         |"history"
+DATA_BROKER_URL             |Data Broker address                                           |"http://data-broker"
+DEVICE_MANAGER_URL          |Device Manager address                                        |"http://device-manager:5000"
+DOJOT_SUBJECT_TENANCY       |Global subject to use when publishing tenancy lifecycle events|"dojot.tenancy"
+DOJOT_SUBJECT_DEVICES       |Global subject to use when receiving device lifecycle events  |"dojot.device-manager.device"
+DOJOT_SUBJECT_DEVICE_DATA   |Global subject to use when receiving data from devices        |"device-data"
+DOJOT_SERVICE_MANAGEMENT    |Global service to use when publishing dojot management events |"dojot-management"
 
 ## Installation
 
-As this service is written in Python 2.7, it is recommended that all packages are
-installed within a virtual environment. This can be created by:
+### Standalone setup
+
+As this service is written in **Python 2.7**, it is recommended that all packages are
+installed within a [virtual environment](https://github.com/pypa/virtualenv). It can be created by:
 
 ```shell
-virtualenv ./venv
-source ./venv/bin/activate
+$ virtualenv ./venv
```

Comment:
`$` is not necessary once the file reader will format it as commands, in this way the user can copy and paste the content without modifications